### PR TITLE
check_ctdb: return WARNING when no script was run, and fix scriptstatus split

### PR DESCRIPTION
**Pull request by sathieu**

### CODE REVIEW

File: ctdb/utils/nagios/check_ctdb [hunk 1]
```diff
@@ -223,6 +223,10 @@ if ($info eq "scriptstatus") {
         if ($result == OK) {
             $result = $np->check_threshold(check => $error_script_count, warning => '0', critical => '0');
         }
+        if ($result == OK and $script_count == 0) {
+            $result = WARNING;
+            $output .= "no script run. ";
```
Comment (sathieu) [inline ctdb/utils/nagios/check_ctdb]: I'm not sure. Should it be `no script ran. `?

**Comment by martin-schwenke** [comment]
I think "no script run" is fine.  :-)

**Comment by martin-schwenke** [comment]
We really need to clean this thing up.  I have a couple of commits at:

  https://git.samba.org/?p=martins/samba.git;a=shortlog;h=refs/heads/ctdb-nagios

However, every time I want to try to test the Nagios plugin I find it maddening (almost as maddening as this GitHub interface!)...  can you please explain what Debian packages I need to install and how to test?

Perhaps we could have this conversation on the samba-technical mailing list?  :-)

**Comment by sathieu** [comment]
I will focus on https://github.com/samba-team/samba/pull/168/ first. This one is low priority for me. OK to discuss this on samba-technical.

**Comment by sathieu** [comment]
Closing. I don't need this anymore ...

**Pull request closed without merging**